### DISCUSSION TITLE
Best service for image gallery iframes

### DISCUSSION THREAD

**Discussion by james-l-smith** [knowledgefutures/pubpub#1582] [Ideas]
Hi All,

I was wondering if anyone had any tips about image gallery services to use in pubs via iframe. It looks like Flickr and Google etc have blocked iframe and only use scripts now, so I'm currently using Albumizr. However, it's pretty basic and doesn't have all of the features I want, so I was hoping for some tips.

All the best,
James

**Comment by james-l-smith** [comment]
On this topic, I've found Albumizr to be better than expected at this task, with the little hack of putting 'full screen' with a link to the full screen url of the album in the iframe's caption (since the album has its own captions within the iframe).

**Reply by writingpoli** [reply]
A not-free but useful service you might consider is [Cloudflare's new Images service](https://developers.cloudflare.com/images/). Fairly cheap, it seems.

**Reply by writingpoli** [reply]
Also, you can always just upload the image to PubPub and use a Text block to insert it (or HTML, if you want).
